### DISCUSSION TITLE
Archive FTC Technology Forum and Sub-Forums

### DISCUSSION THREAD

**Discussion by AlecHub** [FIRST-Tech-Challenge/FtcRobotController#96] [Ideas]
Now that GitHub repos have "**Discussions**" (i.e. Forums) and now that this repo is the permanent repo for the FTC control system, moving forward can we create all new forum topics in this Forum instead of [FTC Technology Forum]( https://ftcforum.firstinspires.org/forum/ftc-technology/)?  

It would be great if everything related to FTC Technology, SDK, Wiki, Issues, Forums, Releases, is in one place.  Consolidating the KB makes things easier to find and easier to cross-reference.

The rest of the forums, "~~Game Q&A~~", "Share your thoughts with FIRST Tech Challenge Staff", etc., can stay on [FTC Forum](https://ftcforum.firstinspires.org).

**Comment by Windwoes** [comment]
One potential concern I have with GitHub discussions/issues is that they don't seem to be well-suited to long threads. For instance, when you have an issue with close to 100 posts, you have to click "show N more posts" multiple times to be able to load the full page. Moreover it seems to take a while to load when doing so. By contrast the forums break up messages into multiple smaller pages.

**Reply by AlecHub** [reply]
> One potential concern I have with GitHub discussions/issues is that they don't seem to be well-suited to long threads. For instance, when you have an issue with close to 100 posts, you have to click "show N more posts" multiple times to be able to load the full page. Moreover it seems to take a while to load when doing so. By contrast the forums break up messages into multiple smaller pages.

On the negative side, posting a message on the FTC Forum is super frustrating:  After composing your message you click on the Preview button to make sure your post will be rendered the way you expect.  The rendering looks perfect in the Preview window so you click on the Post Reply button.  Then more often than not most of your post disappears and you have to type your post all over again, not knowing if your second composition will also disappear.  Very frustrating for experienced teams so I can imagine how much more frustrating it is for newer teams.  

In contrast, it is highly unlikely that your Post will not exactly match the Preview on GitHub.  Also, if you want to improve your post, GitHub allows you to edit it.
